config/database.yml

```diff
@@ -1,25 +1,19 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
   adapter: postgresql
+  host: localhost
   pool: 5
   timeout: 5000
+  username: insert_vps_db_username
+  password: vps_db_pword
```

Comment:
Gotta put this in ENV, probably the username too. Had problem doing that myself though, like using the dotenv/dotenv-deployment gems didn't even work.